config/config.go

```diff
@@ -128,6 +130,29 @@ func (cfg *AuctionTimeouts) validate(errs configErrors) configErrors {
 	return errs
 }
 
+func (data *ExternalCache) validate(errs configErrors) configErrors {
+	if !(data.Host == "" && data.Path == "") {
+		// Either external cache host or path not empty, validate.
+		// Return an error if any of these is true
+		if data.Host == "" && data.Path != "" || data.Host != "" && data.Path == "" {
```

Comment:
Why did you change from returning the first error to returning all errors?

I think this might get confusing. For example, if `strings.ContainsAny(data.Host, "://")` fails then `url.Parse("https://" + data.Host + data.Path)` would also fail.